electron/src/paths.ts

```diff
@@ -26,8 +27,45 @@ const PREV_VER_USER_DATA_MIGRATION_FILE_PATHS = [
   // Acorn 5
   path.join(USER_DATA_PATH, `${MIGRATION_FILE_NAME_PREFIX}8`),
 ]
+
+const PROJECTS_HAPP_PATH = app.isPackaged
+  ? path.join(app.getAppPath(), '../app.asar.unpacked/binaries/projects.happ')
+  : path.join(app.getAppPath(), 'binaries/projects.happ')
+
+const PROFILES_HAPP_PATH = app.isPackaged
+  ? path.join(app.getAppPath(), '../app.asar.unpacked/binaries/profiles.happ')
+  : path.join(app.getAppPath(), 'binaries/profiles.happ')
+
+const DATASTORE_PATH = app.isPackaged
+  ? path.join(USER_DATA_PATH, `databases-${INTEGRITY_VERSION_NUMBER}`)
+  : path.join(USER_DATA_PATH, `databases-${INTEGRITY_VERSION_NUMBER}`)
```

Comment:
```suggestion
const DATASTORE_PATH = path.join(USER_DATA_PATH, `databases-${INTEGRITY_VERSION_NUMBER}`)
```

try to take a closer look at things ;)